migration/k8s.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: "apps/v1"
+kind: "Deployment"
+metadata:
+  name: "<workflow_name>"
+  namespace: "default"
+  labels:
+    app: "<workflow_name>"
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: "<workflow_name>"
+  template:
+    metadata:
+      labels:
+        app: "<workflow_name>"
+    spec:
+      serviceAccountName: <gke_service_account>
+      containers:
+      - name: "psl-to-gmk-1"
```

Comment:
why psl-to-gmk-1?

Reply:
The container needs a name, this is the one that the yaml populated by default